package.json

```diff
@@ -6,8 +6,12 @@
     "examples/*"
   ],
   "scripts": {
-    "lint": "wsrun -m -e -c lint",
-    "audit": "wsrun -m -e -c audit"
+    "lint": "yarn workspaces foreach -v --parallel run lint",
+    "audit": "yarn workspaces foreach -v --parallel run audit",
+    "build:pkgs": "yarn workspaces foreach -v --parallel --from '@ibm-watson/*' run build",
+    "build:apps": "yarn workspaces foreach -v --parallel run build:app",
+    "test": "yarn workspaces foreach -v --parallel run test",
+    "g:eslint": "cd $INIT_CWD && eslint"
```

Comment:
This is a "global" script, which is callable from anywhere in the workspace (currently used in `examples/discovery-search-app/package.json`). This allows us to install `eslint` pkg at the top level, then use in multiple places. Otherwise, yarn v2 would require that each workspace include `eslint` as a dependency -- this would make maintenance cumbersome, as we'd have to update the `eslint` version in multiple places.

See https://yarnpkg.com/getting-started/qa#how-to-share-scripts-between-workspaces.

Reply:
Neat!